deployment/terraform/live/vars.tf

```diff
@@ -57,3 +57,8 @@ variable "rds_master_user_password" {
   description = "Password for the master DB user"
   default     = ""  # will be autogenerated if left blank
 }
+
+variable "rds_engine_version" {
+  description = "PostgreSQL engine verision to target for RDS instances"
+  default     = "10.5"
+}
```

Comment:
Not needed.

Reply:
If I hardcode this value in rds/main.tf someone will have to edit tracked files to bring up an instance targeting a non-`10.5` RDS Snapshot. I thought that this would still provide some flexibility in the iterim until we know that all deployed instances of refinery are up to date.

Reply:
The upgrade operation is atomic 9.3 -> 10.5. Both versions were tested, as well as the upgrade procedure, so there is no need to bother with intermediate versions.
Actually, setting a value here that is different from the one specified in `postgres.pp` will probably result in error anyway.
So, what would be useful is to leave the precise RDS version hardcoded in `rds/main.tf` then pass it to Puppet like `RDS_ENDPOINT_ADDRESS` but for now it can just stay hardcoded in those two places.

Reply:
It is my understanding that the packages provided by the [PostgreSQL Apt repository](https://wiki.postgresql.org/wiki/Apt) don't allow for the installation of specification of minor versions. Currently when one does a: `apt-get install postgresql-<major_version>`, the latest minor version is installed. I think it just happens to be that we have the ability to be more specific when creating an RDS instance.

I will move the variable back to a hardcoded value in `postgres.pp`